docs/bricks_design.rst

```diff
@@ -0,0 +1,602 @@
+
+Bricks design explained
+=======================
+
+This is a runnable tutorial, which means you can copy its fragments 
+to an IPython notebook and run. The import section follows:
+
+    >>> import theano
+    >>> theano.config.compute_test_value = 'raise'
+    >>> from theano import tensor
+
+Please keep in mind that the code in this tutorial was written for illustration
+purposes only. Often only the common case is handled and corner cases are
+ignored.
+
+Introduction
+------------
+
+Deep learning (DL) models implementing using Theano share numerous
```

Comment:
Can you add a disclaimer that you assume the reader to know and understand Theano?